app/src/lib/git/config.ts

```diff
@@ -196,7 +196,7 @@ export async function removeGlobalConfigValue(
     HOME: string
   }
 ): Promise<void> {
-  removeConfigValueInPath(name, null, env)
+  await removeConfigValueInPath(name, null, env)
```

Comment:
I could be wrong but shouldn't these basically be `return`s instead of awaits. This way we aren't "double awaiting" assuming you have awaits on the "caller"s